apps/cowswap-frontend/src/modules/tokensList/pure/TokensVirtualList/index.tsx

```diff
@@ -33,7 +38,11 @@ export function TokensVirtualList(props: TokensVirtualListProps) {
     props
   const { values: balances, isLoading: balancesLoading } = balancesState
 
+  const { account: connectedAccount } = useWalletInfo()
+
+  const isInjectedWidgetMode = isInjectedWidget()
   const isWalletConnected = !!account
+  const isChainIdUnsupported = useIsProviderNetworkUnsupported()
```

Comment:
Same thing, pure component, don't use hooks

Reply:
I will refactor that part along with connecting it to the logic